BUILD.bazel

```diff
@@ -128,6 +128,7 @@ cc_library(
         "include/fixed_containers/fixed_red_black_tree_ops.hpp",
         "include/fixed_containers/fixed_red_black_tree_storage.hpp",
         "include/fixed_containers/fixed_red_black_tree_types.hpp",
+        "include/fixed_containers/fixed_red_black_tree_view.hpp",
```

Comment:
This should be a separate target for reduced invalidation, as the `fixed_red_black_tree` does not need the view to operate at this time.

Reply:
5f1e6a9